src/report/merge.ml

```diff
@@ -0,0 +1,17 @@
+(* This file is part of Bisect_ppx, released under the MIT license. See
+   LICENSE.md for details, or visit
+   https://github.com/aantron/bisect_ppx/blob/master/LICENSE.md. *)
+
+
+let output ~to_file ~coverage_files ~coverage_paths ~expect ~do_not_expect =
+  let coverage =
+    Input.load_coverage
+      ~coverage_files ~coverage_paths ~expect ~do_not_expect in
```

Comment:
i.e. call this with `~expect:[] ~do_not_expect:[]` if you agree that `--expect` and `--do-not-expect` are not needed for `merge`.